requirements.txt

```diff
@@ -1,4 +1,4 @@
-aiomqtt < 2.0
```

Comment:
So I had some difficulty getting everything to install from scratch but I dont remember the exact error or reason I had to do this, I think it could be because of the version thats in pip